metricshub-doc/src/site/markdown/configuration/configure-monitoring.md

```diff
@@ -682,14 +682,16 @@ resourceGroups:
 ```
 ### Programmable Configuration
 
-You can write simple [Velocity scripts](https://velocity.apache.org) to automatically configure your monitoring resources by pulling data from an HTTP portal (like NetBox), reading local files, or parsing JSON content.
+You can write simple [Velocity scripts](https://velocity.apache.org) to automatically configure your monitoring resources by pulling data from an HTTP portal (like NetBox), reading local files, SQL databases, or parsing JSON content.
 
 To fetch and transform this data into valid configuration blocks, use [Velocity Tools](https://velocity.apache.org/tools/3.1/tools-summary.html) in the `.vm` template files located under the `/config` directory, such as:
 
 * `${esc.d}http` for making HTTP requests
 * `${esc.d}file` for reading local files
 * `${esc.d}json` for parsing JSON data
 * `${esc.d}collection` for splitting strings or manipulating collections
+* `${esc.d}sql` for executing SQL queries on a database
+* `${esc.d}env` for retrieving environment variables, typically for accessing sensitive information. Use `${esc.d}env.get("<ENV_VARIABLE_NAME>")`, where `<ENV_VARIABLE_NAME>` is the name of your system environment variable.
```

Comment:
* `${esc.d}env` for retrieving environment variables. Use `${esc.d}env.get("<ENV_VARIABLE_NAME>")`, where `<ENV_VARIABLE_NAME>` is the name of your system environment variable.